Sources/PopOverViewController.swift

```diff
@@ -80,26 +126,57 @@ open class PopOverViewController: UITableViewController, UIAdaptivePresentationC
         
         // If explanation text is coming, display it in two lines
         if (descriptions == nil) {
-            cell = tableView.dequeueReusableCell(withIdentifier: "SingleTitleCell")!
+            let reuseIdentifier = "SingleTitleCell"
+            if let c = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier) {
+                cell = c
+            } else {
+                cell = UITableViewCell(style: UITableViewCellStyle.default, reuseIdentifier: reuseIdentifier)
+            }
             cell.textLabel?.text = title
         } else {
             let description:String? = descriptions?[indexPath.row]
 
             if (description?.count)! > 0 {
-                cell = tableView.dequeueReusableCell(withIdentifier: "SubTitleCell")!
-                
+                let reuseIdentifier = "SubTitleCell"
+                if let c = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier) {
+                    cell = c
+                } else {
+                    cell = UITableViewCell(style: UITableViewCellStyle.default, reuseIdentifier: reuseIdentifier)
+                }
+
                 cell.textLabel?.text = title
                 cell.detailTextLabel?.text = description
             } else {
-                cell = tableView.dequeueReusableCell(withIdentifier: "SingleTitleCell")!
+                let reuseIdentifier = "SingleTitleCell"
+                if let c = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier) {
+                    cell = c
+                } else {
+                    cell = UITableViewCell(style: UITableViewCellStyle.default, reuseIdentifier: reuseIdentifier)
+                }
                 cell.textLabel?.text = title
             }
         }
         
-        if (selectRow == nil) {
+        if !selectedRows.contains(indexPath.row) {
+            cell.textLabel?.textColor = titleColor
+            if indexPath.row < imageNames.count, let imageName = imageNames[indexPath.row] {
```

Comment:
Conditional binding should come first in if statements, so 
```
 if indexPath.row < imageNames.count, let imageName = imageNames[indexPath.row]
```
should be
```
 if let imageName = imageNames[indexPath.row], indexPath.row < imageNames.count
```

Reply:
Array size should be checked first before binding